autoload/airline/themes/seoul256.vim

```diff
@@ -0,0 +1,61 @@
+" Created by JB Kopecky (https://github.com/jbkopecky)
```

Comment:
I believe this was originally based on [junegunn/seoul256.vim](https://github.com/junegunn/seoul256.vim)
Hence I think we need to mention it. 
What do you think?

Reply:
I added a reference to junegunn.